addons/purchase_stock/models/purchase_order.py

```diff
@@ -124,27 +124,25 @@ def _get_action_add_from_catalog_extra_context(self):
             'vendor_suggest_days': self.partner_id.suggest_days,
             'vendor_suggest_based_on': self.partner_id.suggest_based_on,
             'vendor_suggest_percent': self.partner_id.suggest_percent,
-            'po_state': self.state,
         }
 
-    def action_purchase_order_suggest(self):
+    def action_purchase_order_suggest(self, search_domain, section_id):
```

Comment:
We can't change method signature in stable.
We didn't care a lot about this rule lately because after OXP we still got a lot of work to do to clean the 19.0 and it was realistic to expect no one already started to work on 19.0 outside of Odoo, but at some point we will have to care to avoid breaking change in stable again.
Those parameters can be pass in the context (or at least, use a default value so `action_purchase_order_suggest` can still be called with no parameter) but keep this change for master :+1: